packages/concerto-core/test/decoratormanager.js

```diff
@@ -62,16 +63,16 @@ describe('DecoratorManager', () => {
 
     describe('#validate', function() {
         it('should support syntax validation', async function() {
-            const dcs = fs.readFileSync('./test/data/decoratorcommands/web.json', 'utf-8');
+            const dcs = fs.readFileSync(path.join(__dirname+'/data/decoratorcommands/web.json'), 'utf-8');
```

Comment:
Arguments to path.join should be a comma separated list of strings: https://nodejs.org/api/path.html#pathjoinpaths

Reply:
Yeah!
Fixed that.